adflow/pyADflow.py

```diff
@@ -4599,34 +4552,34 @@ def _getDefOptions(self):
             'cflcoarse':[float, 1.0],
             'mgcycle':[str, '3w'],
             'mgstartlevel':[int, -1],
-            'resaveraging':[str,'alternateresaveraging'],
+            'resaveraging':[str, ['alternateresaveraging', 'noresaveraging', 'alwaysresaveraging']],
             'smoothparameter':[float, 1.5],
             'cfllimit':[float, 1.5],
             'useblockettes':[bool, True],
             'uselinresmonitor':[bool, False],
 
-            # Overset Parameters:
+            # Overset Parameters
             'nearwalldist':[float, 0.1],
             'backgroundvolscale':[float, 1.0],
             'oversetprojtol':[float, 1e-12],
             'overlapfactor':[float, 0.9],
             'oversetloadbalance':[bool, True],
             'debugzipper':[bool, False],
-            'zippersurfacefamily':[object, None],
+            'zippersurfacefamily':[(str, type(None)), None],
             'cutcallback':[object, None],
```

Comment:
this could be replaced by `(types.FunctionType, types.BuiltinFunctionType)` I think. Not super important but we are expecting a callable here right?

Reply:
Good call. I will update this. I'm inclined to not include `types.BuiltinFunctionType` though because it really should not be a built-in function.

Reply:
Sure, I wasn't sure what types of functions gets passed in.